src/Templating/Helper/BlockHelper.php

```diff
@@ -134,6 +138,18 @@ public function __construct(
         $this->blockContextManager = $blockContextManager;
         $this->cacheHandler = $cacheHandler;
         $this->stopwatch = $stopwatch;
+
+        // NEXT_MAJOR: remove this check
+        if ($cacheManager !== null) {
```

Comment:
We need to update the signature to allow passing the HttpCacheHandlerInterface as 6th param and Stopwatch as 7th.

Reply:
actually its even more complicated... also the second argument `$cacheBlocks` will disappear